### ISSUE TITLE
Add copy explaining photography containing copywrighted works

### ISSUE THREAD

**Issue by zackkrida** [WordPress/openverse#2807]
## Problem

On occasion CC-licensed works may contain photographic subjects—buildings, artworks, or corporate logos, for example—which are, in fact, copywrighted. 

## Description

We should add language to Openverse which clarifies how to handle these types of work. Here is the proposed copy to add to the "How to use" section below the "Visit the {medias} website" paragraph:

> Photographs may contain copyrighted works (for example, paintings, sculptures or architectural works). The use of photographs depicting copyrighted works may require additional permissions from the copyright holder of those works.

This paragraph should conditionally render _only_ on image results.

**Comment by HarshYadavade** [comment]
I would like to work on this issue

**Comment by dhruvkb** [comment]
I don't think we have a design for this. @fcoveram can confirm but in the meantime I'll apply a "design: needed" label.

@HarshYadavade please pick another issue to work on till we have design mockups. Once we have those, you can work on it.

**Comment by fcoveram** [comment]
Thanks for the ping. I don't think this ticket requires design, as the new paragraph should follow the existing style. However, I will share this ticket in the marketing channel asking for more eyes on the copy.

**Comment by rmartinezduque** [comment]
The suggested text sounds good to me! However, here are a few suggestions for minor edits that I think can enhance its readability and make it easier for users to grasp the message quickly:

> Some photographs might contain copyrighted content, such as paintings, sculptures, or architectural works. Use of such photographs may require additional permissions from the copyright holder of the depicted works.

Or

> Some photographs might contain copyrighted content, such as paintings, sculptures, or architectural works. Using these photographs may require additional permissions from the copyright holder of the depicted works.

**Comment by fcoveram** [comment]
Thanks @rmartinezduque, I like the second option.

> Some photographs might contain copyrighted content, such as paintings, sculptures, or architectural works. Using these photographs may require additional permissions from the copyright holder of the depicted works.

Replacing the `design needed` label with the `ready for work`.

**Comment by lucasferrazlima** [comment]
@dhruvkb @zackkrida  I'm interested in working on this issue. May I take it up? 
Thanks!

**Comment by dhruvkb** [comment]
@lucasferrazlima we're currentl in a frontend code freeze to migrate to Nuxt 3 so frontend contributions cannot be merged. In the meantime would you like to work on something related to the catalog, ingestion server or API instead?

**Comment by lucasferrazlima** [comment]
@dhruvkb Oh got it! Thanks for the update. Yes, i would also like to work on these. I'll check for some other interesting discussions.

**Comment by dhruvkb** [comment]
Thank you very much, I'll update here once the freeze is lifted.

**Comment by zackkrida** [comment]
Hi @lucasferrazlima, our frontend code freeze _is_ lifted if you're interested in revising this work. 

If not, that is also fine, and I can take on this PR this week.

**Comment by lucasferrazlima** [comment]
Hey @zackkrida, thanks for letting me know!

You can take the PR.
